packages/contracts-bedrock/tasks/deposits.ts

```diff
@@ -77,26 +81,39 @@ task('deposit', 'Deposits funds onto L2.')
     )
     console.log(`Got TX hash ${tx.hash}. Waiting...`)
     const receipt = await tx.wait()
-    console.log(
-      `Included in block ${receipt.blockHash} with index ${receipt.logIndex}`
-    )
+    console.log(`Included in block ${receipt.blockHash}`)
 
     // find the transaction deposited event and derive
     // the deposit transaction from it
     const event = receipt.events.find(
       (e: Event) => e.event === 'TransactionDeposited'
     )
-    const l2tx = DepositTx.fromL1Event(event)
     console.log(`Deposit has log index ${event.logIndex}`)
+    const l2tx = DepositTx.fromL1Event(event)
     const hash = l2tx.hash()
     console.log(`Waiting for L2 TX hash ${hash}`)
 
+    let i = 0
     while (true) {
       const expected = await l2Provider.send('eth_getTransactionByHash', [hash])
       if (expected) {
         console.log('Deposit success')
+        console.log(JSON.stringify(expected, null, 2))
         break
       }
+      const postL2Balance = await l2Provider.getBalance(to)
+      if (postL2Balance.gt(preL2Balance)) {
+        console.log(
+          `Unexpected balance increase without detecting deposit transaction`
+        )
+      }
+
+      if (i % 20 === 0) {
+        const block = await l2Provider.getBlock('latest')
+        console.log(`latest block [${block.number}]:${block.hash}`)
```

Comment:
Maybe print a timestamp here to provide some context within the [wall of logs this outputs](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/4423/workflows/2f7ab782-08f1-4a7b-a826-2b7ed011acad/jobs/65188/parallel-runs/0/steps/0-106)

Reply:
Yeah either that or reduce the period between logs